internal/cli/users.go

```diff
@@ -22,8 +66,341 @@ func usersCmd(cli *cli) *cobra.Command {
 	}
 
 	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listUsersCmd(cli))
+	cmd.AddCommand(createUserCmd(cli))
+	cmd.AddCommand(showUserCmd(cli))
+	cmd.AddCommand(deleteUserCmd(cli))
+	cmd.AddCommand(updateUserCmd(cli))
 	cmd.AddCommand(userBlocksCmd(cli))
 	cmd.AddCommand(deleteUserBlocksCmd(cli))
+
+	return cmd
+}
+
+func listUsersCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Args:    cobra.NoArgs,
+		Short:   "List your users.",
+		Long: `List your existing users. To create one try:
+auth0 users create`,
+		Example: `auth0 users list
+auth0 users ls`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var list *management.UserList
+
+			if err := ansi.Waiting(func() error {
+				var err error
+				list, err = cli.api.User.List()
+				return err
+			}); err != nil {
+				return fmt.Errorf("An unexpected error occurred: %w", err)
+			}
+
+			cli.renderer.UserList(list.Users)
+			return nil
+		},
+	}
+
+	return cmd
+}
+
+func createUserCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		Connection 	string
+		Email		string
+		Password	string
+		Username	string
+		Name		string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "create",
+		Args:  cobra.NoArgs,
+		Short: "Create a new user.",
+		Long:  "Create a new user.",
+		Example: `auth0 users create 
+auth0 users create --name "John Doe" 
+auth0 users create -n "John Doe" --email john@example.com`,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			prepareInteractivity(cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
```suggestion
auth0 users create -n "John Doe" --email john@example.com`,
		RunE: func(cmd *cobra.Command, args []string) error {
```

`prepareInteractivity ` was refactored last Friday, now it's on the root command.